addons/crm/views/crm_lead_views.xml

```diff
@@ -731,7 +716,108 @@
+                    <field name="city" optional="hide"/>
+                    <field name="state_id" optional="hide"/>
+                    <field name="country_id" optional="hide"/>
+                    <field name="user_id" widget="many2one_avatar_user" optional="show"/>
+                    <field name="team_id" optional="show"/>
+                    <field name="priority" optional="hide"/>
+                    <field name="activity_ids" widget="list_activity"/>
+                    <field name="activity_user_id" optional="hide" string="Activity by" widget="many2one_avatar_user"/>
+                    <field name="activity_date_deadline_my" string="My Deadline" widget="remaining_days" options="{'allow_order': '1'}"/>
+                    <field name="campaign_id" optional="hide"/>
+                    <field name="medium_id" optional="hide"/>
+                    <field name="source_id" optional="hide"/>
+                    <field name="company_currency" invisible="1"/>
+                    <field name="planned_revenue" sum="Expected Revenues" optional="show" widget="monetary" options="{'currency_field': 'company_currency'}"/>
+                    <field name="stage_id" optional="show" decoration-bf="1"/>
+                    <field name="active" invisible="1"/>
+                    <field name="probability" optional="hide"/>
+                    <field name="company_id" groups="base.group_multi_company" optional="show"/>
+                    <field name="tag_ids" optional="hide" widget="many2many_tags" options="{'color_field': 'color'}"/>
+                    <field name="referred" invisible="1"/>
+                    <field name="message_needaction" invisible="1"/>
+                </tree>
+            </field>
+        </record>
+
+        <record id="crm_case_tree_view_leads2" model="ir.ui.view">
+            <field name="name">crm.lead.tree.lead</field>
+            <field name="model">crm.lead</field>
+            <field name="mode">primary</field>
+            <field name="priority">15</field>
+            <field name="inherit_id" ref="crm.crm_case_tree_view_leads"/>
+            <field name="arch" type="xml">
+                <field name="user_id" position="attributes">
+                    <attribute name="optional">show</attribute>
+                </field>
+                <field name="message_needaction" position="after">
+                    <button name="action_snooze" type="object" string="Snooze 7d" icon="fa-bell-slash" />
+                    <button name="%(crm.action_lead_single_mail)d" type="action" string="Email" icon="fa-envelope"
+                            context="{
+                                        'default_composition_mode': 'comment',
+                                        'default_use_template': True,
+                                    }"
+                    />
+                </field>
+            </field>
+        </record>
+        <record id="crm_case_tree_view_my_activities" model="ir.ui.view">
```

Comment:
Right xml ID is crm_lead_view_list_activities (model + view (telling it is a view record) + list (as in list / form / kanban / ...) + suffix to help (see guidelines for more details).